demo/run_demo.py

```diff
@@ -34,6 +34,13 @@ def supply_chain():
   subprocess.call(write_code_cmd.split())
   copyfile("foo.py", "../functionary_carl/foo.py")
```

Comment:
We had talked about this earlier. If you remember, the problem was that the 2 foo.tar.gz files had different SHA codes. So we decided to make write-code a threshold step.